packages/core/admin/package.json

```diff
@@ -28,7 +28,7 @@
     "develop:webpack": "cross-env NODE_ENV=development webpack serve --config webpack.config.dev.js --progress profile",
     "prepublishOnly": "yarn build",
     "build": "rimraf build && node ./scripts/build.js",
-    "build:mesure": "rimraf build && cross-env MESURE_BUILD_SPEED=true node ./scripts/build.js",
+    "build:measure": "rimraf build && cross-env MEASURE_BUILD_SPEED=true node ./scripts/build.js",
```

Comment:
`MEASURE_BUILD_SPEED` – have you checked this is the spelling in the build script? :)

Reply:
Yeah, the script is looking for MEASURE_BUILD_SPEED ( /packages/core/admin/scripts/build.js line 60), currently it's not working at all